herbert_core/configuration/log_config.m

```diff
@@ -60,8 +60,8 @@
     end
 
     properties(Access=protected, Hidden=true)
-        info_log_print_time_ = 30 % normally info log should be printed
-        %                         % every 30 secodd
+        info_log_print_time_ = 300 % normally info log should be printed
```

Comment:
Why? I understand that this comment wasn't added this PR, but why should the log be printed every X timeperiod?

Reply:
I was working with alignment and found this issue. This number controls log string (`printed N#out of #F`). Dots are printed with every page